falcon/api.py

```diff
@@ -465,12 +465,7 @@ def _get_responder(self, req):
         Note:
             If a responder was matched to the given URI, but the HTTP
             method was not found in the method_map for the responder,
-            the responder callable element of the returned tuple will be
```

Comment:
Would you please revert this change to the docstring? `_get_responder` does not call the responder itself, but just returns it.